projects/canopy/src/lib/content-scroll/content-scroll.component.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+  ElementRef,
+  Renderer2,
+} from '@angular/core';
+
+import { ContentScrollBreakpoints } from './content-scroll.interface';
+
+@Component({
+  selector: 'lg-content-scroll',
+  styleUrls: ['./content-scroll.component.scss'],
+  templateUrl: './content-scroll.component.html',
+  encapsulation: ViewEncapsulation.None,
+})
+export class LgContentSrollComponent {
+  @HostBinding('class') class = 'lg-content-scroll';
+
+  private _scrollContentAt: ContentScrollBreakpoints;
+  @Input()
+  set scrollContentAt(columnsBreakpoint: ContentScrollBreakpoints) {
+    if (columnsBreakpoint === null) {
+      columnsBreakpoint = this._scrollContentAt;
+    }
+
+    this.addContentScrollBreakpoint(columnsBreakpoint);
+    this._scrollContentAt = columnsBreakpoint;
+  }
+  get scrollContentAt(): ContentScrollBreakpoints {
+    return this._scrollContentAt;
+  }
+
+  private _scrollHeight: string;
+  @Input()
+  set scrollHeight(value: string) {
+    if (value === null || value === '') {
+      value = this._scrollHeight;
+    }
+    this.setHeight(value);
+    this._scrollHeight = value;
+  }
+  get scrollHeight(): string {
+    return this._scrollHeight;
+  }
+
+  @Input() listNoIndent = false;
+
+  constructor(private renderer: Renderer2, private hostElement: ElementRef) {
+    this._scrollContentAt = ContentScrollBreakpoints.Small;
+    this._scrollHeight = '40vh';
+
+    this.addContentScrollBreakpoint(this._scrollContentAt);
```

Comment:
As this method gets called in the setter, I'm wondering if we should set it directly?

E.g. `this.scrollContentAt = ContentScrollBreakpoints.Small`

The same thing would go for `this.scrollHeight`.

Or is it done like this for a reason?